instruqt/closing-information-gap/03-additional-information/assignment.md

```diff
@@ -0,0 +1,336 @@
+                Harbor requires a minimum of 2 CPU cores in order to run, and runs best with
+                at least 4 cores. Your current cluster has less than 2 CPU cores available to Kubernetes
+                workloads. Please increase cluster capacity or install into a different cluster.
+              uri: https://goharbor.io/docs/2.8.0/install-config/installation-prereqs/
+          - warn:
+              when: "sum(cpuAllocatable) < 4"
+              message: |-
+                Harbor runs best with a minimum of 4 CPU cores. Your current cluster has less
+                than 4 CPU cores available to run workloads. For the best experience, consider
+                increasing cluster capacity or installing into a different cluster.
+              uri: https://goharbor.io/docs/2.8.0/install-config/installation-prereqs/
+          - pass:
+              message: Your cluster has sufficient CPU resources available to run Harbor
+    - nodeResources:
+        checkName: Cluster memory is sufficient to install and run Harbor
+        outcomes:
+          - fail:
+              when: "sum(memoryAllocatable) < 4G"
+              message: |-
+                Harbor requires a minimum of 4 GB of memory in order to run, and runs best with
+                at least 8 GB. Your current cluster has less than 4 GB available to Kubernetes
+                workloads. Please increase cluster capacity or install into a different cluster.
+              uri: https://goharbor.io/docs/2.8.0/install-config/installation-prereqs/
+          - warn:
+              when: "sum(memoryAllocatable) < 8Gi"
+              message: |-
+                Harbor runs best with a minimum of 8 GB of memory. Your current cluster has less
+                than 8 GB of memory available to run workloads. For the best experience, consider
+                increasing cluster capacity or installing into a different cluster.
+              uri: https://goharbor.io/docs/2.8.0/install-config/installation-prereqs/
+          - pass:
+              message: Your cluster has sufficient memory available to run Harbor
+```
+
+You may want to add these as your first analyzers, since they call attention
+to items that might prevent workloads from running and might explain other
+failed analyzers.
+
+![Adding Cluster Analyzers](../assets/adding-cluster-analyzers.png)
+
+Generate the More Complete Support Bundle
+==========================================
+
+Now that you've enhanced your support bundle with more information about the
+running application, let's generate another bundle and take a look at the
+analyzers.
+
+```
+kubectl support-bundle ./harbor-support-bundle.yaml
+```
+
+You'll see that that your bundle has been collected and get a
+text interface showing the results of the analyzers you added.
+
+![Complete Support Bundle Analysis](../assets/complete-support-bundle-analysis.png)
+
+Note the failure in for the `harbor-jobservice` workload. This may be the cause
```

Comment:
this doesn't show a failure?

Reply:
Good catch. I updated the setup scripts to not show a failure and didn't update the text.